src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -90,12 +91,21 @@ def generate_json_report(self):
             "energy_trade_profile": self.energy_trade_profile
         }
 
+    def _write_none_to_unmatched_loads(self, area):
```

Comment:
I think that it is better to move this function inside the MraketUnmatchedLoads class, since it is relevant only to the unmatched loads.

Reply:
Ok, yes. Will do.